web/src/compose_reply.ts

```diff
@@ -256,7 +256,8 @@ export function quote_message(opts: {
             topic = message.topic;
             stream_id = message.stream_id;
         }
-
+        compose_state.set_is_stream_recipient_dropdown_for_forward_message_opened(true);
+        compose_state.set_is_processing_forward_message(true);
```

Comment:
This still seems very strange. (1) These two variables I think always have the same value, and (2) can't we just directly check whether that `dropdown_widget` is open using JS rather than having a state variable?